### PR TITLE
Added ValidateCheckSum property

### PR DESCRIPTION
**Pull request by BrianFarnhill**

### CODE REVIEW

**Comment by BrianFarnhill** [comment]
@caadam - Feel like a quick code review of something that isn't SharePoint?

File: appveyor.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: 1.0.{build}.0
+version: 1.1.{build}.0
```
Comment (KarolKaczmarek) [inline appveyor.yml]: We are increasing version numbers when we merge into the master, right before the release. Can you please revert the version change?

**Comment by caadam** [comment]
Mate. What's with the xAzureStorage?

Sent from my Windows Phone

File: README.md [hunk 2]
```diff
@@ -35,6 +35,10 @@ Review the "examples" directory in the xAzureStorage resource for some general e
 
 ## Version History
 
+### 1.1.0.0
```
Comment (KarolKaczmarek) [inline README.md]: You've reverted version increament, please also change this to 

### Unreleased

File: appveyor.yml [hunk 3]
```diff
@@ -24,7 +24,7 @@ after_test:
     - ps: |
         Remove-Item (Join-Path "$env:APPVEYOR_BUILD_FOLDER\xAzureStorage\modules\xAzureStorage" "xAzureStorage.pssproj")
         $manifest = Join-Path "$env:APPVEYOR_BUILD_FOLDER\xAzureStorage\modules\xAzureStorage" "xAzureStorage.psd1"
-        (Get-Content $manifest -Raw).Replace("1.0.0.0", $env:APPVEYOR_BUILD_VERSION) | Out-File $manifest
+        (Get-Content $manifest -Raw).Replace("1.1.0.0", $env:APPVEYOR_BUILD_VERSION) | Out-File $manifest
```
Comment (KarolKaczmarek) [inline appveyor.yml]: please revert as well

**Comment by BrianFarnhill** [comment]
@KarolKaczmarek All changes made dude, ready to merge when you are

**Comment by KarolKaczmarek** [comment]
Thanks! Merged

**Pull request merged**